pavelib/assets.py

```diff
@@ -718,6 +705,28 @@ def execute_compile_sass(args):
         )
 
 
+def get_parsed_option(options, name, default=None):
+    """
+    Options to a command are provided as string value. Convert it to list where a list is expected
+    after fetching an option specified by name
+    Args:
+        options: Command line arguments passed via paver command.
+        name: name of option to parse
+        default: if option not in options, return default
+    """
+    option = default
+    try:
+        option = getattr(options, name)
+        if isinstance(option, basestring):
```

Comment:
No, I mean, once you have a value extracted, all it is doing is converting that value to list if it a string. a method named `listfy`  will only be converting str option to list.